susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1223,6 +1223,14 @@
         'PDID' : [-8, 2007], 'BETAPDID' : [2066], 'PKGLIST' : RES8 + RES8_X86,
         'DEST' : '/srv/www/htdocs/pub/repositories/res/8/bootstrap/'
     },
+    'alibaba-2-x86_64': {
+        'PDID': [-21, 1683], 'BETAPDID': [2065], 'PKGLIST': [pkg for pkg in RES7 if pkg not in "python-urllib3"] + RES7_X86,
+        'DEST': '/srv/www/htdocs/pub/repositories/alibaba/2/bootstrap/'
+    },
```

Comment:
When we add this to sumatoolbox later, we may use a different ID for alibaba. Maybe it would be better to remove this entry for now and wait until we really added it.

Reply:
If we are not sure if we'll include it, I'd remove it.

For Amazon and specially Alma, I added it because I am sure we'll add then soon. So when the moment comes, we just need to change the first ID, and that's it (as we'll reuse the RES client tools, which are the second and third IDs)

Reply:
I'd rather keep it because it has the special detail about not making `python-urllib3` part of the bootstrap (it's installed on Alibaba Cloud Linux 2 via pip and conflicts at filesystem level).

Reply:
Well, but we'd keep the Uyuni bootstrap repo, so no problem about missing that :-)

Reply:
I've removed the SUSE Manager entry and I have modified the Uyuni line too. No point in removing python-urllib3 since it is pulled in as a dependency of another package :-/

I will add a known issue to the documentation (there will be an error on bootstrap from a minimal system but the client registration will work). The issue will be gone once we start using the bundle.